docs/configuring-polaris-for-production.md

```diff
@@ -56,7 +56,7 @@ Be sure to secure your metastore backend since it will be storing credentials an
 
 ### Configuring EclipseLink
 
-To use [EclipseLink](https://eclipse.dev/eclipselink/) for metastore management, specify the configuration `metaStoreManager.conf-file` to point to an [EclipseLink `persistence.xml` file](https://eclipse.dev/eclipselink/documentation/2.5/solutions/testingjpa002.htm). This file, local to the Polaris service, will contain information on what database to use for metastore management and how to connect to it.
+To use [EclipseLink](https://eclipse.dev/eclipselink/) for metastore management, specify the configuration `metaStoreManager.conf-file` to point to an [EclipseLink `persistence.xml` file](https://eclipse.dev/eclipselink/documentation/4.0/solutions/solutions.html#TESTINGJPA002). This file, local to the Polaris service, will contain information on what database to use for metastore management and how to connect to it. Check out [metastore doc](./metastores.md) for details.
```

Comment:
I'd remove the paragraph completely, and only keep a link to metastores.md, or move everything from metastores.md to here. I'd prefer the later to avoid jumping between two pages for readers. I admit this will make the page a bit longer, but I think it's fine comparing to jump between two pages while following instructions. WDYT?

Reply:
I feel it makes the page too long since we may add more sample configuration like mysql. I would prefer to simplify this paragraph to have a summary here and add a link.  How do you think?

Reply:
I think it's important to complete the setting by just looking at one page if we really care about how easy developer can setup a production ready Polaris. We can remove unnecessary stuff to make it compact, like putting the MySQL and H2 to separated doc, and only keep Postgres here. With that, it isn't a blocker for this PR. We can always improve on it.

Reply:
Sounds good. I will keep them separate for now.  Feel it would be strange to have Postgres in the main doc and have the others separately. But it's not critical and we can merge later if needed.